string setting_node
int32 robot_state
---
bool set_succeeded
